std_msgs/Header header

ObjectDescription[] other_list
AnnotationMsg[] annotation_list
PedestrianMsg[] pedestrian_list
VehicleMsg[] vehicle_list
TrafficLightMsg[] traffic_light_list